src/constants/labels.json

```diff
@@ -3,7 +3,8 @@
     "welcome": "Willkommen bei Lunes!",
     "selectProfession": "Wähle deinen Beruf!",
     "skipSelection": "Überspringen",
-    "confirmSelection": "Starten"
+    "confirmSelection": "Starten",
+    "searchProfession": "Suche Beruf"
```

Comment:
In my opinion the term "Suche Beruf" sounds a little bit "holperig". How about prompting the user with "Beruf eingeben" or "nach Beruf suchen".